templates/function-base/js/default/package.json

```diff
@@ -5,7 +5,6 @@
         "start": "npx func start"
     },
     "dependencies": {
-        "@microsoft/microsoft-graph-client": "^3.0.0",
```

Comment:
What about ts function?

Reply:
ts function is matched with SDK because SDK upgraded graph library to ^3.0.0.